TU20Bot/Models/AccountModel.cs

```diff
@@ -8,36 +8,36 @@
 
 namespace TU20Bot.Models {
     public class AccountModel {
-        public const int SALT_SIZE = 32;
-        public const int HASH_SIZE = 32;
-        private const int ITERATIONS = 100000;
+        private const int saltSize = 32;
+        private const int hashSize = 32;
+        private const int iterations = 100000;
```

Comment:
The style we're using is here :D https://discord.com/channels/230737273350520834/736741911150198835/737076636725411913

What's confusing to me is that constants throughout the project are using this style already, these variables were an exception.

Reply:
It's probably even better to just set up a linter so we aren't allowed to merge these kinds of mistakes at all.

Reply:
In that case we will need review the style guide as a group this time. When scanning code it helps to have constants visually differentiated from local variables within classes.

Reply:
A lot of people think that... but I'm not sure if that argument holds water.

Do you really care if the `username` in `getUser(username)` is a constant or not? Languages like Swift and Rust are widely adopted and don't make any styling suggestions for constants (they basically look and feel like any other variable). The C# `const` keyword is really just there to make compiler optimization simpler and to avoid people 
mistakenly modifying them at runtime.

Reply:
Please be civil during discussions on public forums.

Reply:
I meant for it to be phrased as a thought experiment 😅 but I can see how it can be construed as a bit passive-aggressive. Sorry...